iroh/src/magicsock/remote_map/remote_state.rs

```diff
@@ -707,7 +715,26 @@ impl RemoteStateActor {
             .map(|daddr| daddr.addr)
             .collect::<BTreeSet<_>>();
 
-        match conn.initiate_nat_traversal_round() {
+        let all_rtts = self.calculate_path_rtts();
+        let relay_mapped_addrs = self.relay_mapped_addrs.clone();
+        let relay_latencies = self.last_report.as_ref().map(|r| r.relay_latency.clone());
+        let rtt_hints = |addr: &SocketAddr| {
+            let addr = relay_mapped_addrs.to_transport_addr(*addr)?;
+            match addr {
+                transports::Addr::Relay(url, _id) => {
+                    // Relay case, grab the latency from net_report
+                    relay_latencies.as_ref().and_then(|l| l.get(&url))
+                }
+                addr @ transports::Addr::Ip(_) => {
+                    // IP case, grab the worst latency we currently have
+                    all_rtts
+                        .get(&addr)
+                        .and_then(|rtts| rtts.iter().max().copied())
```

Comment:
I think this should be `.min(Duration::from_millis(333))` additionally, just so we don't accidentally balloon the PTO in case there was some kind of weird bad RTT in our IP paths.

Reply:
the probes will timeout faster for broken paths, that’s the whole point of this PR